go.mod

```diff
@@ -6,85 +6,93 @@ require (
 	github.com/DATA-DOG/go-sqlmock v1.3.3 // indirect
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/coreos/go-semver v0.3.0
-	github.com/coreos/prometheus-operator v0.34.0
+	github.com/coreos/prometheus-operator v0.38.0
 	github.com/fatih/structtag v1.1.0
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
 	github.com/go-logr/logr v0.1.0
 	github.com/go-logr/zapr v0.1.1
 	github.com/gobuffalo/packr v1.30.1 // indirect
```

Comment:
It seems go tools don't automatically remove indirect dependencies. Can you try removing them and re-running `go mod tidy`?

Reply:
We cannot remove all indirect ones. But I removed all that is possible.